hack/builder/Dockerfile

```diff
@@ -18,21 +15,22 @@ ENV PATH=$PATH:/usr/local/go/bin:/go/bin \
 # Build args for code-generator versions
 ARG K8S_VER
 ARG KUBEOPENAPI_VER
-
-
 # Install code generation tools as root
 RUN go install \
-	k8s.io/code-generator/cmd/deepcopy-gen@${K8S_VER} \
-	k8s.io/code-generator/cmd/defaulter-gen@${K8S_VER} && \
-	go install \
-	k8s.io/kube-openapi/cmd/openapi-gen@${KUBEOPENAPI_VER}
-
+    k8s.io/code-generator/cmd/deepcopy-gen@${K8S_VER} \
+    k8s.io/code-generator/cmd/defaulter-gen@${K8S_VER} && \
+    go install \
+    k8s.io/kube-openapi/cmd/openapi-gen@${KUBEOPENAPI_VER}
+# Install rsync and rsync-daemon (for compatibility across different distributions)
+# In Fedora, rsync includes daemon functionality, but rsync-daemon may exist in other distros
+RUN dnf install -y rsync && \
```

Comment:
Please combine with the RUN command above. I don’t think we need to split the two dnf’s